src/SuperSocket.Channel.Kestrel/KestrelPipeChannel.cs

```diff
@@ -0,0 +1,428 @@
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Net.Sockets;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Connections;
+using Microsoft.Extensions.Logging;
+using SuperSocket.ProtoBase;
+
+namespace SuperSocket.Channel.Kestrel
+{
+    public class KestrelPipeChannel<TPackageInfo>
```

Comment:
I don't see a factory for it or a piece of code instantiate it?

Reply:
How can we use this kind of channel?

Reply:
<img width="1728" alt="image" src="https://github.com/kerryjiang/SuperSocket/assets/39239954/37d313b6-256f-4fa0-a3d1-574039820945">
Look here.